recipes-extended/onie-tools/onie-tools/onie-bisdn-backup

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright (C) 2025 Jonas Gorski of BISDN GmbH
+# Bash script to backup or restore BISDN Linux configuration.
+
+set -e
+
+source /usr/libexec/bisdn/backup.sh
+
+USAGE="Create or restore a backup of BISDN Linux configuration
+usage: $(basename "$0") [OPTIONS]
+
+[OPTIONS]
+-h show this message
+-l list backup contents (show only what would be backed up)
+-b <FILENAME> create a tar.gz configuration backup as <FILENAME>
+-r <FILENAME> restore a tar.gz configuraton backup from <FILENAME>
+
+Example usage:
+$(basename "$0") -b bisdn-linux-backup.tgz
+
+For additional information, see the man page for onie-bisdn-backup(1).
+"
+
+BACKUP_FILE=
+MODE=
+
```

Comment:
Not passing any arguments should give the same result as `-h` (instead of "Please run this program as root" or some such).